app/main.py

```diff
@@ -17,13 +17,13 @@ def battle(knights_config: dict) -> dict:
         )
         knight.get_ready()
         knights_list.append(knight)
-    (lancelot,
-     arthur,
-     mordred,
-     red_knight) = (knights_list[0],
-                    knights_list[1],
-                    knights_list[2],
-                    knights_list[3])
+    knights = [
+        knights_list[0],
+        knights_list[1],
+        knights_list[2],
+        knights_list[3]
+    ]
+    lancelot, arthur, mordred, red_knight = knights
```

Comment:
Noo, the idea is to not use indexes at all. You have a list of knights. You know their order. Thus you can unpack it in 1 line as I mentioned before

Reply:
Yeah, got you, it is done